.github/workflows/commit-linter.yml

```diff
@@ -0,0 +1,11 @@
+name: Lint Commit Messages
+on: [pull_request]
+
+jobs:
+  commitlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: wagoid/commitlint-github-action@v4
```

Comment:
Also include the `commitlint.config.js`, I think there's is one you can copy in one of our private repos.